Userland/Applications/FontEditor/GlyphEditorWidget.cpp

```diff
@@ -31,6 +31,9 @@ void GlyphEditorWidget::set_glyph(int glyph)
 
 void GlyphEditorWidget::paint_event(GUI::PaintEvent& event)
 {
+    if (!m_font)
```

Comment:
Shouldn't that be done in the previous commit? Or at least this commit should appear first.

Reply:
Uhh.. I don't see why that should be done in any other commit?

Reply:
I can totally miss something, but as I currently see it:
[LibGUI: Don't call did_change_font in GlyphMapWidget's constructor](https://github.com/SerenityOS/serenity/pull/18029/commits/72fdd9d27ecf5bfda8de912f72f952444d52ba08) has a side effect: it's "possible" to draw a null font, which result in what I presume is a crash.
[FontEditor: Don't paint in GlyphEditorWidget if m_font is null](https://github.com/SerenityOS/serenity/pull/18029/commits/15ef6332fb3183a5cdaf8f1e20e9d1508d2c45b7) solves the issue.

So the code is partially broken if you checkout 72fdd9d27ecf5bfda8de912f72f952444d52ba08, doing it the other way around or squashing both commits and makes the issue disappear.

Reply:
No, both of these commits make the final commit work without crashes. They are not directly related

Reply:
I just realized that both commit don't touch the same object, my bad :disappointed: